packages/starknet-snap/package.json

```diff
@@ -1,39 +1,51 @@
 {
   "name": "@consensys/starknet-snap",
   "version": "2.8.0",
-  "license": "(Apache-2.0 OR MIT)",
+  "keywords": [],
   "repository": {
     "type": "git",
     "url": "https://github.com/ConsenSys/starknet-snap.git"
   },
+  "license": "(Apache-2.0 OR MIT)",
+  "author": "",
   "main": "./dist/bundle.js",
   "files": [
     "dist/",
     "images/",
     "snap.manifest.json"
   ],
   "scripts": {
-    "build:clean": "yarn clean && yarn build",
+    "allow-scripts": "yarn workspace root allow-scripts",
     "build": "mm-snap build",
-    "serve": "mm-snap serve",
+    "build:clean": "yarn clean && yarn build",
     "clean": "rimraf dist",
-    "start": "mm-snap watch",
-    "test": "yarn run test:unit && yarn run cover:report",
+    "cover:report": "nyc report --reporter=lcov --reporter=text",
     "lint": "yarn lint:eslint && yarn lint:misc --check",
     "lint:eslint": "eslint . --cache --ext js,ts",
     "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write",
     "lint:misc": "prettier '**/*.ts' '**/*.json' '**/*.md' '!CHANGELOG.md' --ignore-path .gitignore",
-    "test:unit": "nyc --check-coverage --statements 80 --branches 80 --functions 80 --lines 80 mocha --colors -r ts-node/register \"test/**/*.test.ts\"",
-    "test:unit:one": "nyc --check-coverage --statements 80 --branches 80 --functions 80 --lines 80 mocha --colors -r ts-node/register",
```

Comment:
can we keep this test:unit:one ? It's handy when working on one test file to not have to run all the tests every time.

Reply:
sure

Reply:
done